content/millennium/dstu2/scheduling/slot.md

```diff
@@ -63,32 +63,36 @@ _Implementation Notes_
-`-location`       | No                         | [`reference`] | A single or comma separated list of Location references. Example: `633867`
-`start`           | Yes when using `slot-type` | [`date`]      | The Slot date-time. Example: `2016`
-[`_count`]        | No                         | [`number`]    | The maximum number of results to be returned per page.
+------------------|----------------------------|---------------|-------------------------------------------------------------------------------------------------------------------------------------
+ `_id`            | Yes or `slot-type`         | [`token`]     | The logical resource id associated with the resource.
+ `slot-type`      | Yes or `_id`               | [`token`]     | A single or comma separated list of appointment types that can be booked into the slot. Example: `http://snomed.info/sct|394581000`
+ `schedule.actor` | No                         | [`reference`] | A single or comma separated list of Practitioner references. Example: `Practitioner/2578010`
+ `-location`      | No                         | [`reference`] | A single or comma separated list of Location references. Example: `633867`
+ `start`          | Yes when using `slot-type` | [`date`]      | The Slot date-time. Example: `2016`
+ [`_count`]       | No                         | [`number`]    | The maximum number of results to be returned per page.
 
 Notes:
 
-- The `slot-type`, `schedule.actor`, and `-location` parameters may be included only once.
+* The `slot-type`, `schedule.actor`, and `-location` parameters may be included only once.
   For example, `-location=633867,4048128` is supported but `-location=633867&-location=4048128` is not.
 
-- `slot-type` is a required search parameter when `_id` is not provided.
+* `slot-type` is a required search parameter when `_id` is not provided.
 
-- When `slot-type` is provided, `start` must be provided.
+* When `slot-type` is provided, `start` must be provided.
   For example, `start=2018&slot-type=http://snomed.info/sct|394581000,http://snomed.info/sct|394602003`
 
-- The `start` parameter may be provided:
-  - once without a prefix or time component to imply a date range. (e.g. `&start=2016`, `&start=2016-07`, `&start=2016-07-04`)
-  - twice with the prefixes `ge` and `lt` to indicate a specific range. The date and prefix pairs must define
+* The `slot-type` parameter may contain standard codes or proprietary codes.
+  * When using standard codes, choose codes bound to Slot.type as noted in the [Terminology Bindings](#terminology-bindings) table above.
+  * When using proprietary codes, the system should be `https://fhir.cerner.com/<your EHR source id>/codeSet/<code set>` (where code set is a Millennium code set) and the code should be a Millennium code value. Example: `https://fhir.cerner.com/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/codeSet/14249|4062744`
```

Comment:
Can we switch this to reflect 'start' needs to exist when slot-type is provided, something like 
`start=2018&slot-type=https://fhir.cerner.com/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/codeSet/14249|4062744`

Also, i think we should document that we only support one slot-type at once, either proprietary or standard.

Reply:
disagree with the first part - this example is only the value of a slot-type parameter. this is not a full blown query. the note about slot-type being provided mandating start must be provided already exists on L81

good catch for the second. update L84 to "* The `slot-type` parameter may contain standard codes or proprietary codes. When using standard codes, proprietary codes may not be provided, and vice versa."

Reply:
good call, fixed - 647ac49d650c225fb918808f2d2df0f5d686d51e

Reply:
ugh, let me fix typos